### PR TITLE
Appletv Login and Axis Login Documentation Added.

### PR DESCRIPTION
**Pull request by AleksaZatezalo**
Working on auxilary scanner docs. Issue https://github.com/rapid7/metasploit-framework/issues/12389. Added documentation for appletv_login.

# Verification

Steps needed to make things work are as follows:
- [ ] Start `msfconsole`
- [ ] run `msf> use auxiliary/scanner/http/appletv_login`
- [ ] run `msf> info -d`
- [ ] Verify that you see proper documentation

### CODE REVIEW

**Comment by jheysel-r7** [comment]
@AleksaZatezalo just took a quick glance and wanted to say great job on these docs files, the formatting is looking great. Just one thing - it seems like you've uploaded two PRs with the same documentation files in each.

File: documentation/modules/auxiliary/scanner/http/appletv_login.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+## Vulnerable Application
+
+This module attempts to authenticate to an AppleTV service with the username, 'AirPlay'.
+The device has two different access control modes: OnScreen and Password.
+The difference between the two is the password in OnScreen mode is numeric-only and four digits long,
+which means when this option is enabled, this option, the module will make sure to cover all of them - from 0000 to 9999.
```
Comment (cdelafuente-r7) [inline documentation/modules/auxiliary/scanner/http/appletv_login.md:6]:
```suggestion
which means when this option is enabled, the module will make sure to cover all of them - from 0000 to 9999.
```

File: documentation/modules/auxiliary/scanner/http/appletv_login.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+The Password mode is more complex, therefore the usual online bruteforce strategies apply.
+
+## Verification Steps
+1. Start msfconsole
+2. Do: `use auxiliary/scanner/http/appletv_login`
+3. Do: set the passwords via the `password` option, or pass a list of passwords via the`pass_file` option. Pass a userlist via "user_list".
+4. Do: `run`
+5. Hopefully you see somthing like this:
```
Comment (cdelafuente-r7) [inline documentation/modules/auxiliary/scanner/http/appletv_login.md:14]:
```suggestion
5. Hopefully you see something like this:
```

File: documentation/modules/auxiliary/scanner/http/appletv_login.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+```
+[+] 127.0.0.1:7000 - Login Successful: admin:adminpassword
+```
+
+## Options
+List each option and how to use it.
```
Comment (cdelafuente-r7) [inline documentation/modules/auxiliary/scanner/http/appletv_login.md:20]:
```suggestion
```

File: documentation/modules/auxiliary/scanner/http/appletv_login.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+### BLANK_PASSWORD
+
+Boolean value on if an additional login attempt should be attempted with an empty password for every user.
```
Comment (cdelafuente-r7) [inline documentation/modules/auxiliary/scanner/http/appletv_login.md:24]:
```suggestion
Set to `true` if an additional login attempt should be made with an empty password for every user.
```

File: documentation/modules/auxiliary/scanner/http/axis_login.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+## Options
+List each option and how to use it.
+
+### BLANK_PASSWORDS
+
+Try blank passwords for all users
+
+### BLANK_PASSWORD
+
+Boolean value on if an additional login attempt should be attempted with an empty password for every user.
```
[the remark this answers is not in the document]
Comment (AleksaZatezalo) [reply documentation/modules/auxiliary/scanner/http/axis_login.md]: Thanks for this @cdelafuente-r7. I believe I have committed all your changes.

**Comment by AleksaZatezalo** [comment]
Can someone merge this change please?

**Pull request merged**